python/ambassador/envoy/v2/v2listener.py

```diff
@@ -746,6 +750,36 @@ def compute_routes(self) -> None:
                                 logger.debug("      %s - %s: drop from %s %s%s",
                                              matcher, action, self.name, hostname, extra_info)
 
+            # If we're on Edge Stack and we don't already have an ACME route, add one.
+            if self.config.ir.edge_stack_allowed and not found_acme:
```

Comment:
This needs to go away -- Edge Stack should inject this, rather than having us have magic Edge Stack functionality in Emissary.